components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -780,6 +784,19 @@ + (CGFloat)defaultHeight {
   return kSpinnerRadius * 2.f;
 }
 
++ (NSArray<UIColor *> *)defaultCycleColors {
```

Comment:
In this case the current implementation has no C-style functions and it does have `+ (CGFloat)defaultHeight`, so I was matching that style.  Looking at the other components, I see we `static inline` for single objects and primitives/structs, but no others for a collection like this.  I'd like to leave it as-is for now and discuss changing it in the future.